### DISCUSSION TITLE
How to print or output information to console with PlusLib with the existence of vtkPlusLogger ?

### DISCUSSION THREAD

**Discussion by 1939938853** [PlusToolkit/PlusLib#940] [Q&A]
Hello,

I'm trying to learn how to use PlusLib. Perhaps the very first thing is to be able  to output (print)  information to console.

It seems that ` std::cout<<"Hello PlusLib\n";`  doesn't work if the vtkPlusLogger is enabled.  I may need to output information to console by vtkIGSIOConsoleOutputWindow, but how?  Or set a logfile and tracing the logfile? Like  `  vtkPlusLogger::Instance()->SetLogFileName(outputFileName.c_str());  `? But the log file seems doesn't exist eventhough I gave a name to it.

Thank you for helping!

**Comment by adamrankin** [comment]
I would recommend using the LOG_INFO and similar macros.

**Comment by 1939938853** [comment]
Hi, I set log level to loginfo and used  LOG_INFO. But there is nothing output to console.  LOG_INFO seems only write message to log-file. Further more when vtkPlusLogger is running, std::cout will not work.
